cloudify-manager-worker/README.md

```diff
@@ -1,28 +1,31 @@
+[//]: # -*- markdown -*-
```

Comment:
Is this needed?

Reply:
This is the same issue as below, fixed now.